content/en/security_platform/cspm/getting_started.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Getting Started with CSPM
+kind: documentation
+further_reading:
+- link: "security_platform/default_rules"
+  tag: "Documentation"
+  text: "Explore default cloud configuration rules"
+- link: "security_platform/cspm/findings"
+  tag: "Documentation"
+  text: "Search and explore CSPM findings"
+- link: "security_platform/cspm/frameworks"
+  tag: "Documentation"
+  text: "Learn about frameworks and industry benchmarks"
+---
+
+{{< site-region region="us3,gov" >}}
+<div class="alert alert-warning">
+Cloud Security Posture Management is not currently available in US1-FED or US3.
+</div>
+{{< /site-region >}}
+
+## Overview
+
+Navigate to the [Getting Started page][1] to configure your environment for scanning.
+
+## Setup
+
+### Secure your cloud environment
+
+This section guides you through collecting resource configuration data from cloud providers. Resource configuration collection allows Datadog to assess your environments against Cloud Configuration rules [Add link].
+
+### Secure your hosts and containers
+
+This section guides you through configuring the Datadog agent to scan your hosts and containers. Our agent allows Datadog to continuously assess the state of your hosts and containers against Runtime Compliance rules [Add link].
```

Comment:
Edit: This section guides you through configuring the Datadog agent to scan your hosts and containers. Our agent allows Datadog to continuously assess the state of your hosts and containers against Infrastructure Configuration rules